### DISCUSSION TITLE
Exemplo de conversão 'inteiro_para_cadeia'

### DISCUSSION THREAD

**Discussion by FigFelipe** [dgadelha/Portugol-Webstudio#261] [Dúvidas / Perguntas] [answered]
Para o código abaixo:

```
inclua biblioteca Tipos
inteiro numero = 1234
escreva(Tipos.inteiro_para_cadeia(numero))
```

Recebo como retorno o seguinte erro:

`Tipos incompatíveis! O parâmetro 'base' da função 'inteiro_para_cadeia' espera uma expressão do tipo 'inteiro' com valor, mas foi passada uma expressão do tipo 'vazio' vazia`

Poderiam nos exemplificar o uso da conversão 'inteiro_para_cadeia'?

**Comment by dgadelha** [comment, marked as answer]
Olá,

É necessário especificar a base do número, nesse caso, pode ser a base decimal (10):

```portugol
programa {
  inclua biblioteca Tipos

  funcao inicio() {
    inteiro numero = 1234
    escreva(Tipos.inteiro_para_cadeia(numero, 10))
  }
}
```
